packages/devextreme/testing/testcafe/tests/editors/slider/slider.ts

```diff
@@ -0,0 +1,22 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`Slider`
+  .page(url(__dirname, '../../container.html'));
+
+test('Slider screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .expect(await takeScreenshot('slider.png', 'html'))
```

Comment:
For multitheme testing you should use testScreenshot() function. You can look it up in other tests.